packages/field/src/FormControlMixin.js

```diff
@@ -53,6 +53,34 @@ export const FormControlMixin = dedupeMixin(
         };
       }
 
+      get label() {
+        return (
+          (this.querySelector('[slot="label"]') &&
+            this.querySelector('[slot="label"]').textContent) ||
+          this.__label
+        );
+      }
+
+      set label(newValue) {
+        const oldValue = this.label;
+        this.__label = newValue;
```

Comment:
we have protected _labelNode but private __label? Shouldn't they have the same access?

Reply:
`label` is the public api, `__label` is just needed for internal use.